erpnext/stock/doctype/packing_slip/packing_slip.py

```diff
@@ -30,6 +31,9 @@ def validate(self):
 		validate_uom_is_integer(self, "stock_uom", "qty")
 		validate_uom_is_integer(self, "weight_uom", "net_weight")
 
+	def on_submit(self):
```

Comment:
Had added this code from bloomstack_core to erpnext where under hooks.py files this part of code runs on event : "on submit". Similarly made the functionality to be run on "on submit" event.